### DISCUSSION TITLE
conInfo funktioniert nicht

### DISCUSSION THREAD

**Discussion by Weseler485** [OpenCCU/OpenCCU#1768] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,
ich bekomme es absolut nicht hin der Funktion conInfo etwas zu entlocken. An vielen Stellen ist diese ja eingesetzt und müsste sehr gesprächig sein. Die Console von Firefox und Chrom bleibt aber leer.

Ich habe echt alles versucht was mir einfiel. Hat jemand DEN Tip wie ich das Dingen zum Reden bekomme??

**Comment by jp112sdl** [comment, marked as answer]
Der WebUI-Url noch `debug=true` anhängen.
z.B. `http://192.168.1.254/pages/index.htm?sid=@NzI2e9hAq6@&debug=true`

Weitere "Entwickler-Schalter" findest hier:
https://github.com/eq-3/occu/blob/master/WebUI/www/rega/pages/index.htm#L110-L116
